api-schema.yml

```diff
@@ -683,6 +683,7 @@ components:
       - sureadhere
       - api
       - slack
+      - connect_messaging
```

Comment:
We can also call this "connectid". I like "connect_messaging" a bit better though